### ISSUE TITLE
Check that waynergy works in cosmic-comp 

### ISSUE THREAD

**Issue by WatchMkr** [pop-os/cosmic-comp#450]

**Comment by ids1024** [comment]
This should work, with the right permissions to access `/dev/uinput`. Though this isn't really the best way to handle synergy use-cases on Wayland - https://github.com/r-c-f/waynergy?tab=readme-ov-file#uinput-everyone-else

Waynergy also doesn't seem to have any support for acting as a server: https://github.com/r-c-f/waynergy/issues/57

The correct way now would be the [Remote Desktop](https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.impl.portal.RemoteDesktop.html) and [Input Capture](https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.InputCapture.html) portals, which use libei. Input-leap has Wayland support using these. At least in the long term, these portals to be the correct way to handle synergy-type use cases across Wayland compositors.

https://github.com/Smithay/smithay/pull/1388 has an initial version of the compositor support needed for the remote desktop portal. The Input Capture protocol can also use reis to implement the ei protocol, but still needs work to integrate in Smithay and cosmic-comp.

**Comment by hkdb** [comment]
Hi there! I am actually the one who casually mentioned this to Carl at LinuxFestNW. Just wanted to stop by here and confirm that Waynergy does work with Cosmic. After mentioning it, I decided to test it out myself and it works just fine. My test was with Pop! OS 22.04 and I just installed cosmic-session via apt.

@ids1024 You are absolutely right. Waynergy is not the best way to approach it but it seems to be the best temp workaround I can find for a mixed environment (with the server running x11) at least until input-leap makes enough progress which will probably  be a while.

Can't wait for Cosmic ALPHA! Loving what you guys have done so-far!

Cheers!
